truss/templates/server.Dockerfile.jinja

```diff
@@ -4,13 +4,8 @@
 
 {%- set requires_live_reload = config.live_reload and not config.docker_server %}
 
-{%- if requires_live_reload %}
```

Comment:
Mentioned in PR description, but we no longer need this multi-stage build since `uv` manages python versions + venvs quickly. Even though installing the python version is _slightly_ slower (~5s) than pulling from the `-slim` image, we more than make up for that via `uv pip install` for the control server requirements.